src/Components/Person.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const myСouple = sex === 'm' ? 'wife' : 'husband';
+  const maritalStatus = isMarried
+    ? `My ${myСouple}'s name is ${partnerName}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      <p className="Person__partner">
+        {maritalStatus}
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  isMarried: false,
+  age: null,
```

Comment:
Use conditional rendering instead of showing such values
![image](https://user-images.githubusercontent.com/37161315/144828735-31007785-7606-48f9-9a0b-946672ca8b47.png)